packages/gestalt/src/TileData.test.js

```diff
@@ -0,0 +1,26 @@
+// @flow strict
+import { create } from 'react-test-renderer';
+import TileData from './TileData.js';
+
+describe('TileData', () => {
```

Comment:
These tests are pretty minimal. Can we get better coverage of the various UI states of this component?